Assignment/Assignment/SampleData.cs

```diff
@@ -1,30 +1,46 @@
-﻿using System;
-using System.Collections.Generic;
+﻿namespace Assignment;
 
-namespace Assignment
+public class SampleData : ISampleData
 {
-    public class SampleData : ISampleData
-    {
-        // 1.
-        public IEnumerable<string> CsvRows => throw new NotImplementedException();
+    // 1.
+    public IEnumerable<string> CsvRows => File.ReadLines(@"People.csv").Skip(1);

```

Comment:
```suggestion
    public IEnumerable<string> CsvRows => File.ReadLines("People.csv").Skip(1);

```
You shouldn't need `@` here